sumologic/resource_sumologic_monitors_library_monitor.go

```diff
@@ -588,6 +600,26 @@ func resourceSumologicMonitorsLibraryMonitorRead(d *schema.ResourceData, meta in
 		return nil
 	}
 
+	fgpResponse, fgpErr := c.GetCmfFgp(fgpTargetType, monitor.ID)
+	if fgpErr != nil {
+		/*
+        errCode         |  len  | logic                   |
+        --------------------------------------------------|
+        server_error    |   0   | return err at Get       |
+        server_error    |   1   | warn; return err at Set |
+        not_enabled     |   0   | warn                    |
+        not_enabled     |   1   | warn; return err at Set |
+        */
```

Comment:
We can remove the comment here. There is no "len(...)" in the if condition here.